### PR TITLE
8.0 Release

### PR DESCRIPTION
**Pull request by jennifer-shehane**
Scheduled for July 19, 2021

- [x] Changelog
- [x] Migration Guide
- [x] Client Certificates https://github.com/cypress-io/cypress-documentation/pull/4000
- [x] default —headless for `cypress run` https://github.com/cypress-io/cypress/pull/17309
- [x] Change to default screen size https://github.com/cypress-io/cypress/pull/17309
- [x] Default chrome browser DPR of 1x on headless https://github.com/cypress-io/cypress/pull/17377
- [x] Remove support for older browsers https://github.com/cypress-io/cypress/pull/17355
- [x] Run Cypress as node process https://github.com/cypress-io/cypress-documentation/pull/4017

### CODE REVIEW

File: content/_changelogs/8.0.0.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  browsers. This behavior can be overridden through the
+  [browser launch API](/api/plugins/browser-launch-api#Override-the-device-pixel-ratio).
+  Addresses [#17375](https://github.com/cypress-io/cypress/issues/17375).
+- Cypress now enforces version checks for browser launching and will error
+  during `cypress run` and not allow opening the browser in `cypress open` when
+  attempting to open unsupported browser versions. Cypress supports Chrome >=
+  64, Firefox >= 86, and Edge >= 79. Addressed in
+  [#17355](https://github.com/cypress-io/cypress/pull/17355).
+- When `any` type value is returned from the last function, then the argument
+  will no longer be wrapped with `JQuery`. Fixes
```
Comment (cowboy) [inline content/_changelogs/8.0.0.md:31]: I was having a little trouble understanding this note. This text seems more clear to me (but if it's not accurate, we can skip it)

```suggestion
- No longer wrap returned `any` value from Chainable with `JQuery`. Fixes
```

File: content/_changelogs/8.0.0.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  [#16669](https://github.com/cypress-io/cypress/issues/16669).
+- The `Cypress.ConfigOptions`, `Cypress.ResolvedConfigOptions` and
+  `Cypress.RuntimeConfigOption` types have been updated so that `ConfigOptions`
+  match the JSON schema. Addressed in
+  [#17251](https://github.com/cypress-io/cypress/pull/17251).
+
+**Features:**
+
+- You can now configure certificate authority (CA) and client certificates to
+  use within tests on a per-URL basis via a `clientCertificates` . See
```
Comment (cowboy) [inline content/_changelogs/8.0.0.md:41]:
```suggestion
  use within tests on a per-URL basis via a `clientCertificates` configuration option. See
```

**Review by cowboy — CHANGES_REQUESTED** [review]

File: content/_changelogs/8.0.0.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+in more detail and how to change your code to migrate to Cypress 8.0.**
+
+- When running `cypress run` previous to 8.0, some browsers would launch headed
+  while others were launched headless by default. Cypress now runs all browsers
+  during `cypress run` as headless by default. Addresses
+  [#16832](https://github.com/cypress-io/cypress/issues/16832).
+- The default screen size when running a headless browser has been reverted back
+  to 1280x720 pixels (pre 7.0 behavior). Addresses
+  [#16853](https://github.com/cypress-io/cypress/issues/16853).
+- When running the `—headless` Chrome browser via `cypress run`, the device
```
[the remark this answers is not in the document]
Comment (jennifer-shehane) [reply content/_changelogs/8.0.0.md:20]: Yah, I think my IDE auto-merged this into a em dash after typing 🙃

File: content/_changelogs/8.0.0.md (hunk 1 above)
Comment (jennifer-shehane) [reply content/_changelogs/8.0.0.md:31]: Yeah, this is a not great description, I can try to make it clearer. Basically some things were typed as jQuery objects when they are not necessarily jQuery objects

**Review by flotwig — APPROVED** [review]

**Pull request merged**